src/status_im/contexts/chat/group_create/view.cljs

```diff
@@ -0,0 +1,106 @@
+(ns status-im.contexts.chat.group-create.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [quo.theme]
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [status-im.common.floating-button-page.view :as floating-button-page]
+            [status-im.constants :as constants]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.debounce :as debounce]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [utils.responsiveness :as responsiveness]))
+
+(defn avatar
+  [{:keys [customization-color]}]
+  [rn/pressable {:style {:width 88 :margin-top 12 :margin-left 20}}
+   [hole-view/hole-view
+    {:holes [{:y            (- 80 32)
+              :x            (+ (- 80 32) 8)
```

Comment:
maybe add some consts to understand what these numbers are

Reply:
mhm, it's not like these numbers will be reused to use constants, it's just how hole view works, you have to give some magic numbers to cut the hole